### PR TITLE
10.0 mig stock picking package preparation 

### PR DESCRIPTION
**Pull request by archetipo**
migrate:
 stock_picking_package_preparation
 stock_picking_package_preparation_line

### CODE REVIEW

File: stock_picking_package_preparation/model/stock_picking_package_preparation.py [hunk 1]
```diff
@@ -1,23 +1,8 @@
 # -*- coding: utf-8 -*-
```
Comment (eLBati) [inline stock_picking_package_preparation/model/stock_picking_package_preparation.py:1]: @archetipo dir  `model` should be removed

File: stock_picking_package_preparation/view/stock_picking_package_preparation_view.xml [hunk 2]
```diff
@@ -1,106 +1,102 @@
 <?xml version="1.0" encoding="utf-8"?>
```
Comment (eLBati) [inline stock_picking_package_preparation/view/stock_picking_package_preparation_view.xml]: @archetipo dir `view` should be removed

**Review by eLBati — CHANGES_REQUESTED** [review]

**Comment by archetipo** [comment]
@eLBati thnaks! done.

File: stock_picking_package_preparation/README.rst [hunk 3]
```diff
@@ -38,6 +40,12 @@ finalized yet (no quants), but you can verify the operations in the
 Eventually, when you click on **Done**, all the pickings will be
 transfered and the package operations will be performed.
 
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/154/10.0
+.. repo_id is available in https://github.com/OCA/maintainer-tools/blob/master/tools/repos_with_ids.txt
+.. branch is "10.0" for example
```
Comment (eLBati) [inline stock_picking_package_preparation/README.rst]: Please remove comment

Comment (tafaRU) [reply stock_picking_package_preparation/README.rst]: @eLBati, done.

**Review by eLBati — CHANGES_REQUESTED** [review]
I tried the following steps:
 - Create a new package preparation
 - Add a line in 'details' tab, selecting a product
 - Click 'put in pack'

Get 'Cannot find a location for Delivery Orders'

It seems this use case is not working in 9.0 also.
While it works in 8.0

**Comment by tafaRU** [comment]
@eLBati, please see https://github.com/OCA/stock-logistics-workflow/pull/271/commits/c8f7a11e4203ed23bc3ca4eafb236b3aa852815a

**Review by eLBati — APPROVED** [review]
Thanks

**Comment by eLBati** [comment]
Moving to https://github.com/OCA/stock-logistics-workflow/pull/318
Please close this one

**Pull request closed without merging**